src/components/CountrySelector/style.scss

```diff
@@ -121,33 +122,35 @@ $_sdk-option-border-bottom: 0;
   width: 100%;
 }
 
-.custom__input {
-  background-color: transparent;
-  position: relative;
-}
-
 .custom__hint {
   color: #b1b4b6;
   position: absolute;
 }
 
-.custom__input--default {
-  @extend %_sdk-caret-icon;
-  padding: $_sdk-input-horizontal-padding $_sdk-input-icon-padding
-    $_sdk-input-horizontal-padding $_sdk-input-vertical-padding;
+.custom__input {
+  background-color: transparent;
+  position: relative;
 }
 
 .custom__input--focused {
   outline-offset: 0;
   box-shadow: inset 0 0 0 2 * $unit;
 }
 
+/* If showAllValues is disabled, change class to .custom__input--default & add @extend %_sdk-caret-icon; */
```

Comment:
When is `showAllValues` disabled?

Reply:
Never for us now with this change. Just added the note as a reminder of the relationship to the classes.